packages/medusa/src/repositories/price-list.ts

```diff
@@ -1,5 +1,111 @@
-import { EntityRepository, Repository } from "typeorm"
+import { flatten, groupBy, map } from "lodash"
+import {
+  Brackets,
+  EntityRepository,
+  FindManyOptions,
+  FindOperator,
+  OrderByCondition,
+  Repository,
+} from "typeorm"
 import { PriceList } from "../models/price-list"
+import { PriceListStatus, PriceListType } from "../types/price-list"
 
+type CustomOptions = {
+  select?: FindManyOptions["select"]
+  where?: FindManyOptions["where"] & {
+    status?: FindOperator<PriceListStatus>[]
+    type?: FindOperator<PriceListType>[]
+  }
+  order?: OrderByCondition
+  skip?: number
+  take?: number
+}
+
+type FindOptions = CustomOptions
```

Comment:
why does that type exists? we could directly use the `CustomOptions` instead no?